lib/components/Hidden/Hidden.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component, ReactNode, ReactType } from 'react';
+import Box from '../Box/Box';
+
+export interface HiddenProps {
+  children: ReactNode;
+  hideFor: 'mobile' | 'desktop';
+  component?: ReactType;
+}
+
+export default class Hidden extends Component<HiddenProps> {
+  static displayName = 'Hidden';
+
+  render() {
+    const { children, component, hideFor } = this.props;
+
+    return (
+      <Box
+        display={[
+          hideFor === 'mobile' ? 'none' : 'inline',
```

Comment:
Whats the reason for falling back to `inline` on mobile?

Reply:
so children is the one decide the display style `display: xxx`, if falling back to `block`, children couldn't override it? thoughts?

Reply:
We had to pass something there and `inline` is something.  If the child is block it will still behave as expected.

Better ideas gratefully accepted 😅